0.7.0: can't link without pkg-config

With a very simple test binary calling `notify`, and setting `SYSTEMD_LIBS` and `SYSTEMD_LIB_DIR`, I get:
```
  = note: /usr/lib/gcc/x86_64-pc-linux-gnu/10.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /home/gbr/devel/rust-test/target/debug/deps/rust_test-0eecc3be087929c5.477j6v1y2yw2wxdu.rcgu.o: in function `systemd::daemon::notify':
          /home/gbr/.cargo/registry/src/github.com-1ecc6299db9ec823/systemd-0.7.0/src/daemon.rs:257: undefined reference to `sd_notify'
```

--- a/libsystemd-sys/build.rs
+++ b/libsystemd-sys/build.rs
@@ -57,11 +57,11 @@ fn main() {
         Some(libs) => {
             //let libs = libs.expect(&format!("non utf-8 value provided in {}", lib_var));
             for lib in libs.into_string().unwrap().split(':') {
-                println!("cargo:rustc-link={}", lib);
+                println!("cargo:rustc-link-lib={}", lib);
             }
         }
         None => {
-            println!("cargo:rustc-link={}", name);
+            println!("cargo:rustc-link-lib={}", name);
         }
     }
 }

